tonic/src/status.rs

```diff
@@ -505,11 +505,12 @@ impl Status {
         Status {
             code,
             message: message.into(),
-            details: details,
-            metadata: metadata,
+            details,
+            metadata,
         }
     }
 
+    #[allow(clippy::wrong_self_convention)]
```

Comment:
wut....

Reply:
yeah, apparently it wants the method to be called `into_http`

Reply:
Yeah, I guess its right actually...